mmpose/models/heads/voxelpose_head.py

```diff
@@ -0,0 +1,158 @@
+# ------------------------------------------------------------------------------
+# Adapted from
+# https://github.com/microsoft/voxelpose-pytorch/blob/main/lib/models
+# Original licence: MIT License.
+# ------------------------------------------------------------------------------
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class CuboidCenterHead(nn.Module):
+    """Get results from the 3D human center heatmap.
```

Comment:
If possible, please add more descriptions of the behavior of this head, or refer to the corresponding part of the paper. The current docstring seems too simple to help users understand how to use this head.

Reply:
Ok